e2e/testcases/reconciler_manager_test.go

```diff
@@ -235,7 +235,7 @@ func validateRepoSyncDependencies(nt *nomostest.NT, ns, rsName string) []client.
 
 	repoSyncRB := &rbacv1.RoleBinding{}
 	setNN(repoSyncRB, client.ObjectKey{
```

Comment:
nit: this setNN func seems a little silly when it takes 4 lines, when the alternative setters only take 2 lines. I think it only really makes sense to use if the NN variable is pre-made and re-used.

Reply:
Done